components/Modals/BuyingGuideModal.tsx

```diff
@@ -178,11 +178,21 @@ export default function BuyingGuide() {
         <StepCard
           index={3}
           iconPath="/uniswap.svg"
-          title={"Connect wallet on Uniswap"}
-          content={`Choose WalletConnect and click `}
+          title={
+            isSequenceWallet
+              ? `Connect wallet on Uniswap`
+              : `Buy Glo through Uniswap`
+          }
+          content={
+            isSequenceWallet
+              ? `Choose WalletConnect and click `
+              : `Connect your wallet and click \"Swap\"`
+          }
           action={() => {
-            window.open("https://app.uniswap.org/", "_blank");
-            setIsUniswapStepDone(true);
+            isSequenceWallet
+              ? window.open("https://app.uniswap.org/", "_blank") &&
```

Comment:
this can easily backfire, when the code is adjusted to function that returns void instead of Window reference